Project.toml

```diff
@@ -19,3 +19,4 @@ Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [compat]
 Clipper = "0.6.0"
+FileIO = "1.11"
```

Comment:
I think there should be a compat bound for FileIO, but I'm wondering if this is too aggressive? Is it possible that `FileIO = "1"` would work with these changes? I'm not suggesting an exhaustive search, but maybe check v1, v1.4, and v1.6 (the FileIO docs suggest that 1.6 should work)

Reply:
1.6 works, 1.5 does not. `1.6 - 1` sets the half-closed interval `[1.6, 2.0)` which I think is what we want